include/dali/core/dynlink_cufile.h

```diff
@@ -62,7 +62,7 @@ class CufileError : public std::runtime_error {
         return "cufile: invalid pointer memory type";
       case CU_FILE_CUDA_POINTER_RANGE_ERROR:
         return "cufile: pointer range exceeds allocated address range";
-      case CU_FILE_CUDA_CONTECUFILE_OPT_MISMATCH:
+      case CU_FILE_CUDA_CONTEXT_MISMATCH:
```

Comment:
We still use old headers in our build. Please revert for now.

Reply:
wouldn't it be better to upgrade the header to 0.95? I think its much cleaner

Reply:
Will do